Getting-Started.adoc

```diff
@@ -57,61 +61,75 @@ helm search riff -l
 
 == [[current]]Install the current riff release
 
-=== Install riff using a Kubernetes cluster supporting LoadBalancer
+Starting with the 0.0.4 version we have separated the Kafka installation from the riff chart. This requires an extra step of installing Kafka in order yo use riff. You can use the single-node Kafka chart provided by riff or the three-node kafka/zookeeper service provided by the Kubeapps.
 
-Install the riff helm chart as follows:
+=== Install Kafka chart
 
-For an install with default configuration and no RBAC enabled use:
+We provide a lightweight single node Kafka installation in the `riffrepo/kafka` chart. This works well for development purposes.
+
+Create the `riff-system` namespace:
 
 [source, bash]
 ----
-helm install riffrepo/riff --version 0.0.3 --name demo
+kubectl create namespace riff-system
 ----
 
-[NOTE]
-====
-For a cluster that has RBAC enabled add `--set create.rbac=true`:
+Install the kafka/zookeeper service provided by riff using:
 
 [source, bash]
 ----
-helm install riffrepo/riff --name demo --set create.rbac=true
+helm install --name transport --namespace riff-system riffrepo/kafka
 ----
-====
 
-[WARNING]
+[TIP]
 ====
-The 0.0.3 version of riff needs a fix to the RBAC configuration that is provided in version `0.0.3-rbac` of the chart. Use the following command to install riff `0.0.3` with RBAC enabled:
+Alternatively, install the three-node kafka/zookeeper service provided by the Kubeapps `incubator/kafka` chart using:
 
+[source, bash]
 ----
-helm install riffrepo/riff --version 0.0.3-rbac --name demo
+helm install --name transport --namespace riff-system incubator/kafka
 ----
+
+Just be aware that this chart requires significantly more resources to run.
 ====
 
-=== Install riff using a Kubernetes cluster not supporting LoadBalancer (e.g. minikube)
+=== Install riff using a Kubernetes cluster supporting LoadBalancer
 
-For `NodePort` service types (e.g. running on Minikube) configure the httpGateway to use `NodePort`:
+Install the riff helm chart as follows:
+
+For an install with default configuration and no RBAC enabled use:
 
 [source, bash]
 ----
-helm install riffrepo/riff --version 0.0.3 --name demo --set httpGateway.service.type=NodePort
+helm install riffrepo/riff --version 0.0.4 --name demo --set create.rbac=false
 ----
 
 [NOTE]
 ====
-For a cluster that has RBAC enabled add `--set create.rbac=true`:
+For a cluster that has RBAC enabled use `--set create.rbac=true`:
```

Comment:
Isn't that the default?

Reply:
Tes , it is now, maybe we should just remove it and add a comment that create.rbac=true is the new default

Reply:
I think so, yes